etc/ci.sh

```diff
@@ -8,26 +8,20 @@
 
 set -ex
 
-if [[ $TEST_SUITE = 'makemanuals' && $TRAVIS_OS_NAME = 'linux' ]]
-then
-    make manuals
-    cat doc/*/make_manuals.out
-    if [ `cat doc/*/make_manuals.out | grep -c "manual.lab written"` != '3' ]
-    then
-        echo "Build failed"
-        exit 1
-    fi
-else
-    if [ ! -f  tst/${TEST_SUITE}.g ]
-    then
-        echo "Could not read test suite tst/${TEST_SUITE}.g"
-        exit 1
-    fi
-
-    if [[ x"$ABI" == "x32" ]]
-    then
-        sh bin/gap.sh tst/${TEST_SUITE}.g
-    else
+case $TEST_SUITE in
+    makemanuals)
+        if [[ $TRAVIS_OS_NAME = 'linux' ]]
+        then
+            make manuals
+            cat doc/*/make_manuals.out
+            if [ `cat doc/*/make_manuals.out | grep -c "manual.lab written"` != '3' ]
```

Comment:
I  don't know, but this must've been there before, copied and pasted through the centuries. I can change it though while I am at it.